src/ui/server/helpers/mappings/map-countries-select.test.ts

```diff
@@ -0,0 +1,46 @@
+import { CisCountry } from '../../../types';
+import { mapCountriesSelect } from './map-countries-select';
+
+describe('server/helpers/mappings/map-countries-select', () => {
+  const mockCountries = [
+    {
+      marketName: 'Abu Dhabi',
+      isoCode: 'XAD',
+      shortTermCoverAvailabilityDesc: 'A',
+      ESRAClasificationDesc: 'B',
+      NBIIssue: 'C',
+    },
+    {
+      marketName: 'Algeria',
+      isoCode: 'DZA',
+      oecdRiskCategory: 2,
+      shortTermCoverAvailabilityDesc: 'D',
+      ESRAClasificationDesc: 'E',
+      NBIIssue: 'F',
+    },
+  ] as Array<CisCountry>;
+
+  describe('getSupportedCurrencies', () => {
```

Comment:
There's no function called `getSupportedCurrencies` in this file

Reply:
Also this doesn't reflect what the code is actually doing `it('should only return supported currencies (GBP, EUR, USD)', () => {`